spec/factories/billing_contracts.rb

```diff
@@ -0,0 +1,17 @@
+FactoryBot.define do
+  factory :billing_contract do
+    contract_num { Faker::Number.number(digits: 12) }
+    is_active { true }
```

Comment:
i think you don't need add here is_active property, because i is default on database level